react/features/toolbar/components/Toolbar.web.js

```diff
@@ -0,0 +1,409 @@
+/* @flow */
+
+declare var AJS: Object;
```

Comment:
please add the "global' comment too, that way we'll remember to get rid of them eventually.